conf/configuration.go

```diff
@@ -254,6 +268,30 @@ func Load() {
 	}
 }
 
+// parseIniFileConfiguration is used to parse the config file when it is in INI format. For INI files, it would
+// require a nested structure, so we need to merge the nested [default] section into the root level.
+func parseIniFileConfiguration() {
+	cfgFile := viper.ConfigFileUsed()
+	if filepath.Ext(cfgFile) == ".ini" {
```

Comment:
Do we need to handle different casing here?

Reply:
Good call, fixed!